playbooks/roles/aws_digitalmarketplace_api/tasks/main.yml

```diff
@@ -33,3 +33,18 @@
   register: digitalmarketplace_api_env_stack
   tags:
     - digitalmarketplace-api
+
+- name: RDS dev access security group rules ({{dev_access_state}})
+  cloudformation:
+    stack_name: "digitalmarketplace-api-{{ name_suffix }}-dev-access"
+    state: "{{ dev_access_state }}"
+    region: "{{ aws_region }}"
+    disable_rollback: false
+    template: ../cloudformation_templates/aws_digitalmarketplace_api_rds_dev_access.json
+    template_parameters:
+      CidrIp: "{{ user_ip }}"
+      RDSSecurityGroup: "{{ digitalmarketplace_api_env_stack.stack_outputs.RDSSecurityGroup }}"
+  register: digitalmarketplace_api_rds_dev_access_stack
```

Comment:
I'm trying to use a simple consistent naming scheme for now, so staying away from abbreviations.  This way the vars that end up in common/user var files hopefully will be self-descriptive. But I do agree that a terse convention might be better if we come up with a good one.